test/e2e/adminUI/tests/group005Fields/code/uiTestCodeField.js

```diff
@@ -1,46 +1,8 @@
-module.exports = {
-	before: function (browser) {
-		browser
-		browser.app = browser.page.app();
-		browser.signinPage = browser.page.signin();
-		browser.listPage = browser.page.list();
-		browser.initialFormPage = browser.page.initialForm();
-
-		browser.app.navigate();
-		browser.app.waitForElementVisible('@signinScreen');
-
-		browser.signinPage.signin();
-		browser.app.waitForElementVisible('@homeScreen');
-	},
-	after: function (browser) {
-		browser.app.signout();
-		browser.end();
-	},
-	'Code field should be visible in initial modal': function (browser) {
-		browser.app
-			.click('@fieldListsMenu')
-			.waitForElementVisible('@listScreen')
-			.click('@codeListSubmenu')
-			.waitForElementVisible('@listScreen');
-
-		browser.listPage
-			.click('@createFirstItemButton');
+var fieldTests = require('../commonFieldTestUtils.js');
 
-		browser.app
-			.waitForElementVisible('@initialFormScreen');
-
-		browser.initialFormPage.section.form.section.codeList.section.name
-			.verifyUI();
-
-		browser.initialFormPage.section.form.section.codeList.section.fieldA
-			.verifyUI();
-	},
-	// UNDO ANY STATE CHANGES -- THIS TEST SHOULD RUN LAST
-	'restoring test state': function (browser) {
-		browser.initialFormPage.section.form
-			.click('@cancelButton');
-
-		browser.app
-			.waitForElementVisible('@listScreen');
-	},
+module.exports = {
+	before: fieldTests.beforeUI,
```

Comment:
@webteckie Calling `fieldTests.beforeUI` in place of `fieldTests.beforeUX` for `url` results in the following: 
```
Running:  Url field can be filled via the initial modal
? TypeError: Cannot read property 'expect' of undefined
    at Object.Url field can be filled via the initial modal (D:\Documents\Development\GitHub\keystone\test\e2e\adminUI\tests\group005Fields\commonFieldTestUtils.js:88:5)
    at Module.call (D:\Documents\Development\GitHub\keystone\node_modules\nightwatch\lib\runner\module.js:63:34)
    at D:\Documents\Development\GitHub\keystone\node_modules\nightwatch\lib\runner\testcase.js:97:29
    at _fulfilled (D:\Documents\Development\GitHub\keystone\node_modules\q\q.js:834:54)
    at self.promiseDispatch.done (D:\Documents\Development\GitHub\keystone\node_modules\q\q.js:863:30)
    at Promise.promise.promiseDispatch (D:\Documents\Development\GitHub\keystone\node_modules\q\q.js:796:13)
    at D:\Documents\Development\GitHub\keystone\node_modules\q\q.js:556:49
    at runSingle (D:\Documents\Development\GitHub\keystone\node_modules\q\q.js:137:13)
    at flush (D:\Documents\Development\GitHub\keystone\node_modules\q\q.js:125:13)
    at _combinedTickCallback (node.js:370:9)
    at process._tickCallback (node.js:401:11)

FAILED:  1 assertions failed (17ms)
```